pkg/option/config.go

```diff
@@ -1205,7 +1205,7 @@ type DaemonConfig struct {
 	LibDir           string     // Cilium library files directory
 	RunDir           string     // Cilium runtime directory
 	NAT46Prefix      *net.IPNet // NAT46 IPv6 Prefix
-	Device           string     // Receive device
+	Devices          []string   // bpf_netdev device
```

Comment:
Changing from 1 device to multiple devices requires a change in the flag name. please bring this up in the next weekly meeting